source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,91 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
+
+### Important Upgrade Notes
+ - Removed the deprecated Insights feature.
+ - Mattermost Boards and various other plugins have transitioned to being fully community supported. See this [forum post](https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669) for more details.
+
+**IMPORTANT:** If you upgrade from a release earlier than v8.1, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Improvements
+
+#### User Interface (UI)
+ - The number of channel members is now shown in the **Browse channels** modal.
+ - An error is now displayed if a post edit history fails to load.
+ - Added functionality to bulk mark a whole channel category as read.
+ - Removed Boards product tour code.
+ - Replaced Gfycat with Giphy in the gif picker.
+ - Pre-packaged Calls version v0.19.0.
+ - Updated Focalboard plugin version to 7.11.3.
+ - Pre-packaged Playbooks version 1.38.1.
+ - Upgraded prepackaged Zoom plugin to v1.6.2.
+ - Upgraded prepackaged Antivirus plugin version to 1.0.0.
+
+#### Administration
+ - API examples are now updated to reflect latest Go API conventions, deprecating older code samples.
+ - Updated the public server module version to v0.0.8.
+ - Added a ``Post Action`` plugin hook to allow plugins to register new items in the post menu.
+ - Added a ``Post Editor Action`` plugin hook to allow plugins to register new items in the post editor menu.
+ - Improved logging on plugin initialization, activation, and removal.
+ - Removed the deprecated ``ManifestExecutables`` struct.
+ - Removed the deprecated ``UserAuth.Password`` field.
+ - Remote users are no longer counted as part of the license.
+ - Improved data retention logs.
+ - Removed ``/opengraph`` endpoint as it was unused.
+ - Transitionally prepackaged plugins are now installed to the filestore for continuity when a future release stops prepackaging those plugins.
+ - Removed the deprecated ``Manifest.RequiredConfig`` field.
+ - Added a ``NotificationWillBePushed`` plugin hook invoked before the push notification is processed and sent to the notification service. Plugins may modify or reject the push notification.
+ - Added a `SendPushNotification` plugin api method which allows plugins to send push notifications to a specific user's mobile sessions.
+ - Disabled ``PluginSettings.EnableRemoteMarketplace`` functionality.
+
+### Bug Fixes
+ - Fixed the error returned by ``PUT /api/v4/channels/{channelid}`` when the provided name already existed in the team.
+ - Fixed an issue where CRLF line endings passed to mmctl commands were not being stripped from commands.
+ - Fixed an issue where text copied from Microsoft OneNote is pasted as an image.
+ - Fixed an issue preventing successful activation of trial licenses.
+ - Fixed an issue where a custom group wouldn't get marked as a mention if it was not part of the webapp's local state.
+ - Fixed several issues with loading of licenses.
```

Comment:
It seems this was reverted from 9.0? https://github.com/mattermost/mattermost/pull/24501